modules/entity_version_workflows/tests/Functional/EntityVersionNodeRevisionTest.php

```diff
@@ -35,6 +35,11 @@ class EntityVersionNodeRevisionTest extends BrowserTestBase {
     'entity_version_workflows_example',
```

Comment:
This test contains a usage of `drupalPostForm` which is deprecated and we need to change it.